packages/babel-make-styles/README.md

```diff
@@ -0,0 +1,49 @@
+# @fluentui/babel-make-styles
+
+A Babel plugin that performs build time transforms for [`@fluentui/react-make-styles`](../react-make-styles).
+
+## Install
+
+```bash
+yarn add @fluentui/babel-make-styles
+```
+
+## Usage
+
+`.babelrc`
+
+```json
```

Comment:
perhaps a `js` example too ?

Reply:
If it worth, I can add examples of usage via CLI and Node, like there:
https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types#via-babelrc-recommended

Reply:
Resolving for now, let me know if you have any suggestions.